README.md

```diff
@@ -31,6 +31,7 @@ ___
     - [How can i look into the config of my Palworld container?](#how-can-i-look-into-the-config-of-my-palworld-container)
     - [Im seeing S\_API errors in my logs when i start the container?](#im-seeing-s_api-errors-in-my-logs-when-i-start-the-container)
     - [Im using Apple silicon type of hardware, can I run this?](#im-using-apple-silicon-type-of-hardware-can-i-run-this)
+    - [I have changed the setting but seems like not affect the server.](#i-have-changed-the-setting-but-seems-like-not-affect-the-server)
```

Comment:
"I have changed the settings, but some changes are un-affected in the server"

Sounds/reads better

Reply:
```suggestion
    - [I have changed the settings, but some changes are un-affected in the server?](#i-have-changed-the-settings-but-some-changes-are-un-affected-in-the-server)
```